source/mongorestore.txt

```diff
@@ -706,6 +706,13 @@ Options
    To use :option:`--preserveUUID <mongorestore --preserveUUID>`, you
    must also include the :option:`--drop <mongorestore --drop>` option.
 
+   .. note::
+
+      If ``mongorestore`` uses the 
+      :option:`--preserveUUID <mongorestore--preserveUUID>` option and
+      does not find a UUID for a collection in the restore data,
+      ``mongorestore`` creates a collection with a new UUID. 
```

Comment:
[question] Is it creating a new collection with all the data from the UUID-less collection or is it just generating a UUID for that collection?

Reply:
My understanding (from looking at the ticket and the linked tickets) is that it is creating a new collection with all the data from the UUID-less collection but I will check in with Tim!

Reply:
It's for that collection. Maybe change "creates a collection with a new UUID" to "creates **_the_** collection with a new UUID" to make that clearer?

Reply:
Done!